classes/Rest/Controllers/SummaryControllerProvider.php

```diff
@@ -115,7 +114,7 @@ public function getPortlets(Request $request, Application $app)
 
                     $queryConfig = json_decode($query['config']);
 
-                    if (!$queryConfig->featured) {
+                    if (isset($queryConfig->featured) && !$queryConfig->featured) {
```

Comment:
The code should only include charts that have featured set to true. Otherwise they should be skipped.

```suggestion
                    if (!isset($queryConfig->featured) || !$queryConfig->featured) {
```